### ISSUE TITLE
Problem with using json.jsx in my scripts

### ISSUE THREAD

**Issue by yahiaboudah** [indiscripts/extendscript#3]
When I include json.jsx in my script, and pass an object to stringify using JSON(myObject); it gives me this error:

Exception has occurred.
Error Code# 24: Object is not a function @ file '/c/Projects/extendscript-python-hook/json.jsx' [line:1415, col:1]

and when I use the lave functions: JSON.lave(myObject), I get this:

Exception has occurred.
Error Code# 24: JSON.lave is not a function @ file '/c/Projects/extendscript-python-hook/json.jsx' [line:1415, col:1]

Any idea on how to fix this? other JS json parsers/stringifiers aren't working properly, so I need this urgently. Thank you.

**Comment by indiscripts** [comment]
Hi @yahiaboudah 

Uneasy to help you without seeing the outer context, but I suspect you may have a global `JSON` object declared _before_ mine. So perhaps `$.global.JSON` already exists, and then the `json.jsx` code won't override it, due to the below condition

~~~~
if( !$.global.hasOwnProperty('JSON') )
{

// . . .

}
~~~~

Question: does the [`test.jsx`](https://github.com/indiscripts/extendscript/blob/master/JSON/test.jsx) script properly work?

**Comment by yahiaboudah** [comment]
You were right. So I fixed this with a quick hack instead of "removing" the old JSON obj in my global scope, I did this instead:
if( true )
{

// . . .

}
I had to do this once.
I tried declaring JSON with some value before your JSON script, then deleting it. Setting it to null gives the same previous error. But eventually:

delete(JSON);
if( !$.global.hasOwnProperty('JSON') )
{

// . . .

}

worked perfectly.
